### PR TITLE
Update composition.md

### PR DESCRIPTION
**Pull request by Sammuel09**
I made the following edits to this page:
Removed repeated sentence.
Corrected spelling

### CODE REVIEW

File: docs/guide/advanced/composition.md [hunk 1]
```diff
@@ -413,7 +413,6 @@ In this example, the definition of resources is separated from i18n custom block
 
 ### Shared locale messages for components
 
-In Leacy API mode, shared locale messages are used in components with the `sharedMessages` option.
```
Comment (kazupon) [inline docs/guide/advanced/composition.md:416]: Ah, 
This message is a NOTE to users using legacy mode. So please don't delete it; keep it as a NOTE using Vitepress custom containers syntax. 🙏 
https://vitepress.dev/guide/markdown#custom-containers

**Review by kazupon — CHANGES_REQUESTED** [review]
Thank you for your contribution!
I've just reviewd your PR
Please check it!

File: docs/guide/advanced/composition.md (hunk 1 above)
Comment (Sammuel09) [reply docs/guide/advanced/composition.md:416]: oh my.
my apologies. 
It was meant to be a correction on the spelling for `Legacy` instead of `Leacy`

**Comment by Sammuel09** [comment]
Closing this PR as I have made another pr to contribute the fix.
thank you @kazupon

**Pull request closed without merging**